google/cloud/commerce/doc/main.dox

```diff
@@ -41,6 +41,6 @@ application.
 - @ref commerce-env - describes environment variables that can configure the
   behavior of the library.
 
-[cloud-service-docs]: https://cloud.google.com/commerce
+[cloud-service-docs]: https://cloud.google.com/marketplace/docs/reference/consumerprocurement/rest
```

Comment:
Normally we link a "guide" page vs. a reference page.  I cannot find a good intro to the API, but this is better than the REST reference guide:

https://cloud.google.com/marketplace/docs/commitment-api-purchasing

Or we could just use that the API team is recommending:

https://cloud.google.com/marketplace/docs/

It may get better over time.

Reply:
I left a note on the bug about the doc, hopefully someone can change it to something better, sticking with the API team recommendation